doc/advanced-configuration/outgoing-connections.md

```diff
@@ -206,3 +207,49 @@ Make sure Cassandra is running and then run MongooseIM in live mode:
 
 If no errors occurred and your output is similar to the one above then your MongooseIM and Cassandra nodes can communicate over SSL.
 
+## ElasticSearch connection setup
+
+A connection pool to ElasticSearch can be configured as below:
+
+```erlang
+{outgoing_pools, [
+ {elastic, global, elasticsearch, [], []}
+]}.
+```
+
+MongooseIM uses [inaka/tirerl](https://github.com/inaka/tirerl) library to communicate with ElasticSearch.
+This library starts pool of worker by its own so:
+
+* number of workers (which is 50)
+* `call_timeout` (inifinity)
+* worker selection strategy (`available_worker` or what's set as `default_strategy` of `worker_pool` application)
+
+are not possible to change via `WPoolOpts`.
+
+Only `ConnectionOpts` can be set and in them the following options can be added (as `{key, value}` pairs):
```

Comment:
Only configurable options are `ConnectionOpts` and here you can add (as `{key, value}` pairs):